cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -35,6 +35,8 @@ const (
 	AwsProviderName = "aws"
 	// BaiducloudProviderName gets the provider name of baiducloud
 	BaiducloudProviderName = "baiducloud"
+	// DigitalOceanProviderName gets the provider name of baiducloud
```

Comment:
Typo here, baiducloud should be digitalocean

Reply:
Thanks, fixed it 👍